Samples~/LootLockerExamples/Scripts/PlayerFiles.cs

```diff
@@ -0,0 +1,254 @@
+                {
+                    informationText.text = "Guest session started" + "\n";
+                    GetPlayerFiles();
+                }
+                else
+                {
+                    informationText.text = "Error" + response.Error + "\n";
+                }
+            });
+        }
+
+        public void GetPlayerFiles()
+        {
+            // Get a list of the files that the current player has
+            LootLockerSDKManager.GetAllPlayerFiles((response) =>
+            {
+                if (response.success)
+                {
+                    informationText.text += "Got player files" + "\n";
+                    availablePlayerFilesText.text = "Available files: \n";
+                    for (int i = 0; i < response.items.Length; i++)
+                    {
+                        // Write name and ID of the files to the UI
+                        availablePlayerFilesText.text += "Name:"+response.items[i].name+"\n";
+                        availablePlayerFilesText.text += "ID:" + response.items[i].id + "\n";
+                        availablePlayerFilesText.text += "Purpose:" + response.items[i].purpose + "\n";
+                        availablePlayerFilesText.text += "Size:" + response.items[i].size + "\n";
+                        availablePlayerFilesText.text += "------------------" + "\n";
+                    }
+                }
+                else
+                {
+                    informationText.text += "Error" + response.Error + "\n";
+                }
+            });
+        }
+
+        public void GetPlayerFileContent()
+        {
+            // You get the URL of the file from LootLocker
+            // You need to download the fl yourself from LootLocker.
+            // You can use the IEnumerator Download() method below
+            LootLockerSDKManager.GetPlayerFile(int.Parse(fileIdDownloadInput.text), (response) =>
+            {
+                if (response.success)
+                {
+                    informationText.text += "Got player file" + "\n";
+                    fileIdUploadInput.text = response.id.ToString();
+                    StartCoroutine(Download(response.url, (fileContent) =>
+                    {
+                        informationText.text += "Got player file content" + "\n";
+                        playerFilesTextContent.text = fileContent;
+                    }));
+                }
+                else
+                {
+                    informationText.text += "Error" + response.Error + "\n";
+                }
+            });
+        }
+
+        // Used for downloading files from a URL
+        IEnumerator Download(string url, System.Action<string> fileContent)
+        {
+            UnityWebRequest www = new UnityWebRequest(url);
+            www.downloadHandler = new DownloadHandlerBuffer();
+            yield return www.SendWebRequest();
+
+            if (www.result != UnityWebRequest.Result.Success)
```

Comment:
Less amusingly, UnityWebRequest does not have .result prior to Unity 2020. See LootLockerBaseServerAPI.cs#118 for an example on how we've done this before.